utils/utils.h

```diff
@@ -157,6 +158,22 @@ struct gbConf {
   char volServer[HOST_NAME_MAX];
 };
 
+# define GB_XDATA_MAGIC_NUM 0xDEAD2019DAED
```

Comment:
As asked in my previous review, please consider changing this to something else :D

Reply:
Sure :-)